users/tests/test_views.py

```diff
@@ -136,4 +136,36 @@ def test_post_login_invalid_email_fails(self):
         response = self.client.post(reverse('login'), creds)
         self.assertIn('email', response.json())
         self.assertEqual('Enter a valid email address.', response.json().get('email')[0])
+
+
+class UserprofileTestCase(TestCase):
+    """ Test for user profile details
+    """
+    def setUp(self):
+        self.Client = Client()
+        user = User.objects.create_user(**user_data)
+        self.client.login(email=user_data['email'], password=user_data['password'])
+
+    def test_profile_detail_successful(self):
+        response = self.client.get(reverse('user-auth'))
+        self.assertEqual(response.status_code, 200)
```

Comment:
i said we don't based our test on status code ONLY. check if the details that has been rendered really belongs to the correct user